### ISSUE TITLE
Questions on 3rd party widgets within the CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#11089]
## Description
There are a few 3rd party widgets within the CMS that are throwing Axe accessibility errors.  I am wondering if we have any control over these. My first instinct is no we cannot change anything here. If we cannot, we have the ability to exclude these from the cypress-axe scans so they don't stop PRs from being merged.

Can you confirm if we can or cannot edit the following?:

1. Within the KB, there is a "Contact help desk" widget in the bottom right hand corner.  This element has ID of `jsd-widget`. 
   1. Would it be possible to add a `title` attribute to the `<iframe>` tag in order to provide this element with an accessible name?
   2. ![image](https://user-images.githubusercontent.com/106678594/194112136-d614ef17-b8cd-49b2-9292-76784e169714.png)
2. On Publications nodes, the editors has the option to add an uploaded video. When added, the video is an generates an iframe. 
   1. Would it be possible to add a `title` attribute to the `<iframe>` tag in order to provide this element with an accessible name?
   2. ![image](https://user-images.githubusercontent.com/106678594/194124344-5bbcefa7-a79f-47c6-8c05-a694587ebf99.png)
3. On Vet Center - community access point nodes, there is a google map with a marker in the Geolocation field.  The marker is an image without an `alt` attribute.
   1. Would it be possible to add a `alt` attribute to the `<img>` marker?
   2. ![image](https://user-images.githubusercontent.com/106678594/194126246-2ef76f76-a456-4cb8-87a1-3af67df97ab1.png)

## Acceptance Criteria
- [x] Determine feasibility of editing each of the iframes listed in the description

**Comment by edmund-dunn** [comment]
@laflannery 
For 1 and 3 those aren't ours. 1 belongs to Atlassian and 3 belongs to Leaflet.js. 
2 we can modify because there is a hook_preproccess frunction we can use.
